index.js

```diff
@@ -1,57 +1,65 @@
+class Methods {
+  // constructor() {}
 
-  listOfBooks = JSON.parse(localStorage.getItem('books'));
-  // eslint-disable-next-line eqeqeq
-  listOfBooks = listOfBooks.filter((book) => book.serialNumber != remSerial);
+  static add({ title, author }) {
+    // eslint-disable-next-line no-new
+    new Book({ title, author });
+    localStorage.setItem('books', JSON.stringify(Book.listOfBooks));
+  }
 
-  localStorage.setItem('books', JSON.stringify(listOfBooks)); location.reload();
-}
+  static remove(event) {
+    const remSerial = event.target.parentElement.id;
+    // eslint-disable-next-line eqeqeq
+    Book.listOfBooks = Book.listOfBooks.filter((book) => book.serialNumber != remSerial);
 
-function display() {
-  listOfBooks = JSON.parse(localStorage.getItem('books')) || [];
-  const booksContainer = document.getElementById('display-books');
-  // eslint-disable-next-line no-restricted-syntax
-  for (const book of listOfBooks) {
-    booksContainer.innerHTML += `
-        <div id='${book.serialNumber}'>
-            <p>${book.title}</p>
-            <p>${book.author}</p>
-            <button type="button" class="removebtn" onclick='removeBook(event)'> Remove </button>
-            <hr>
-        </div> 
-        `;
+    localStorage.setItem('books', JSON.stringify(Book.listOfBooks)); location.reload();
   }
-}
 
-function handleAdd(event) {
-  event.preventDefault();
-  const titleInput = document.querySelector('input[placeholder="Title"]');
-  const authorInput = document.querySelector('input[placeholder="Author"]');
-  if (!(titleInput.value.length < 3 || authorInput.value.length < 3)) {
-    addBook(titleInput.value, authorInput.value, listOfBooks);
-    location.reload();
-  } else {
-    const form = document.querySelector('form');
-    const errorDiv = document.createElement('div');
-    form.prepend(errorDiv);
-    errorDiv.innerHTML = `
+  static display() {
+    const booksContainer = document.getElementById('display-books');
+    // eslint-disable-next-line no-restricted-syntax
+    for (const book of Book.listOfBooks) {
+      booksContainer.innerHTML += `
+          <div id='${book.serialNumber}'>
+              <p>${book.title}</p>
+              <p>${book.author}</p>
+              <button type="button" class="removebtn" onclick='Methods.remove(event)'> Remove </button>
+              <hr>
+          </div> 
+          `;
+    }
+  }
+
+  static handleAddtion(event) {
+    event.preventDefault();
+    const titleInput = document.querySelector('input[placeholder="Title"]');
+    const authorInput = document.querySelector('input[placeholder="Author"]');
+    if (!(titleInput.value.length < 3 || authorInput.value.length < 3)) {
+      Methods.add({ title: titleInput.value, author: authorInput.value });
+      location.reload();
+    } else {
+      const form = document.querySelector('form');
+      const errorDiv = document.createElement('div');
+      form.prepend(errorDiv);
+      errorDiv.innerHTML = `
```

Comment:
- [ ] Please try to avoid manipulating the DOM inside the class. Let's try to fix it by creating a function outside the class and manipulating the DOM for the required result in this function.